### ISSUE TITLE
[1.18.2] Crash/Compatibility Issue

### ISSUE THREAD

**Issue by MC-Valdreth** [SammySemicolon/Malum-Mod#187]
Not sure who's end this is on, but the game crashes when using Malum with the newest version of Create. Both could be used together up to Create version 0.5.0i

Reproduction Steps:
1. Install specified Mods/Versions
2. Launch Minecraft
3. Attack anything
4. Game crashes

Crash Report: https://pastebin.com/CdF1E6ah
latest.log: https://pastebin.com/APFu6bz7

Forge Version: 40.2.4
Minecraft Version: 1.18.2
Malum Version: [1.4.4.3](https://www.curseforge.com/minecraft/mc-mods/malum/files/4368497)

Other Mods Installed: 
Create Version: [1.18.2 v0.5.1b](https://www.curseforge.com/minecraft/mc-mods/create/files/4550986)
Lodestone Version: [1.3](https://www.curseforge.com/minecraft/mc-mods/lodestone/files/4353749)
Curios API (Forge) Version: [5.0.9.0](https://www.curseforge.com/minecraft/mc-mods/curios/files/4418032)

**Comment by IttoSanzzo** [comment]
Hey, I just updated create, and I found this error. This is because in this last create update, they changed a lot of terminology and locations in their mod classes, and Malum used one for potato shooter. Solving this is very, very simple, but as the mod is licensed, I can't do it... 

oh, I just went to look, and it looks like Sammy already made the necessary corrections in the mod's src, which means he should update soon with the problem solved... well, apparently, he already fixed it a while ago  (3 weeks), but anyway, just wait a bit.
